frontend/public/components/replication-controller.jsx

```diff
@@ -35,6 +38,16 @@ const Details = ({obj: replicationController}) => {
           </ResourceSummary>
         </div>
         <div className="col-md-6">
+          <dt>Status</dt>
+          <dd>
+            { _.get(replicationController, ['metadata', 'annotations', 'openshift.io/deployment.cancelled']) ?
```

Comment:
noticed that this block is used `workload-table.tsx` as well. We should extract the logic into a separate component so we dont repeat ourselves.